### ISSUE TITLE
Playground block: Mark code editor input field as readonly when it is read only

### ISSUE THREAD

**Issue by brandonpayton** [WordPress/playground-tools#331]
From @alexstine [here](https://github.com/WordPress/Learn/issues/2352#issuecomment-2240790504):

> For the read-only files, awesome that the button communicates that but the input field does not. Think you can open an upstream PR to get aria-readonly or HTML5 readonly support?

**Comment by brandonpayton** [comment]
@alexstine this turned out to be fairly straightforward as the inconsistency was due to a bug in the Playground block itself. No upstream work appears to be necessary. This should be fixed by #337 which is included in the latest (v0.2.12) release. At this moment, the Learn site still needs updated to that version though.

**Comment by adamziel** [comment]
@brandonpayton is this one now resolved, since #337 is merged?

**Comment by brandonpayton** [comment]
This was [tested by someone with a11y expertise](https://github.com/WordPress/Learn/issues/2352#issuecomment-2332416410) and can now be marked as resolved.
